public/app/features/alerting/unified/components/rule-editor/TemplateStep/TemplateStep.service.ts

```diff
@@ -0,0 +1,12 @@
+import { api } from 'app/percona/shared/helpers/api';
+
+import { NotificationListResponse } from './TemplateStep.types';
+
+export const AddAlertRuleModalService = {
+  async notificationList(): Promise<string[]> {
+    const {
+      alertmanager_config: { receivers },
+    } = await api.get<NotificationListResponse, {}>(`/graph/api/alertmanager/grafana/config/api/v1/alerts`);
```

Comment:
https://github.com/airbnb/javascript#strings--quotes a small remark more related to the linter, it seems there is no ${} here and there is no need to use back quotes